app/styles/pages/public.scss

```diff
@@ -12,3 +12,13 @@
   display: flex;
   flex-wrap: wrap;
 }
+
+.rightside-menu {
+  overflow-x: hidden;
+}
+
+@media screen and (min-width: 769px) {
+  .public-container {
+    max-width: 100vw;
```

Comment:
Max width was the issue with the container which was crossing the width of the device.